ministryofjustice/analytics-platform-ops
# Configure fluentd to ship logs to per-env elasticsearch indexes

## What

As part of a move towards a single logging stack for all environments, this change configures all fluentd instances to send logs to a per-environment elasticsearch index, named `logstash-$ENV`.

## How to review

1. Perform a Helm upgrade of the `fluentd` and `rstudio` charts in the dev environment
2. Check fluentd container logs to confirm it has successfully connected to elasticsearch: `$ kubectl logs $FLUENTD_POD_NAME -n kube-system`
3. Check Kibana and verify that logs are present under the `logstash-$ENV` index
